exercises/practice/reverse-string/.meta/src/ReverseString.example.elm

```diff
@@ -0,0 +1,6 @@
+module ReverseString exposing (reverse)
+
+
+reverse : String -> String
+reverse str =
+    String.foldl String.cons "" str
```

Comment:
Why not `String.reverse`?

Reply:
I can switch it to String.reverse, but I wanted to make sure it was solvable other ways. This is a practice exercise so my view is that the example is primarily there for CI purposes. For a concept exercise, I'd use String.reverse since mentors would see the code.

That said, I can change it since I've proven at least to myself there are other solutions to the problem.